librz/debug/dsession.c

```diff
@@ -251,6 +251,7 @@ RZ_API bool rz_debug_session_add_mem_change(RzDebugSession *session, ut64 addr,
 static bool serialize_register_cb(void *db, const ut64 k, const void *v) {
 	RzDebugChangeReg *reg;
 	RzVector *vreg = (RzVector *)v;
+	char tmpbuf[16];
```

Comment:
since there is the `0x` this should be at least 19. set this to 32 so it is mem-aligned.

Reply:
Done